hardware test diffs are failing

```
a = 2.0, b = 4.085951747899065, error = 0.1
    def _assert_approx(a: float, b: float, error: float):
        """assert `b` is within +/- `error` of `a`"""
        # pylint: disable=invalid-name
>       assert (a - error) <= b <= (a + error)
E       assert 4.085951747899065 <= (2.0 + 0.1)
tests/hardware_tests/test_voltage_rw.py:62: AssertionError
```
```
FAILED tests/hardware_tests/test_voltage_rw.py::test_differential_rw_adc_1[DiffMode.DIFF_2-1.0]
FAILED tests/hardware_tests/test_voltage_rw.py::test_differential_rw_adc_1[DiffMode.DIFF_2-2.0]
FAILED tests/hardware_tests/test_voltage_rw.py::test_differential_rw_adc_1[DiffMode.DIFF_2-3.0]
FAILED tests/hardware_tests/test_voltage_rw.py::test_differential_rw_adc_1[DiffMode.DIFF_2-4.0]
FAILED tests/hardware_tests/test_voltage_rw.py::test_differential_rw_adc_2[DiffMode.DIFF_2-1.0]
FAILED tests/hardware_tests/test_voltage_rw.py::test_differential_rw_adc_2[DiffMode.DIFF_2-2.0]
FAILED tests/hardware_tests/test_voltage_rw.py::test_differential_rw_adc_2[DiffMode.DIFF_2-3.0]
FAILED tests/hardware_tests/test_voltage_rw.py::test_differential_rw_adc_2[DiffMode.DIFF_2-4.0]
```

--- a/src/test_edgepi/integration_tests/test_dac/test_dac.py
+++ b/src/test_edgepi/integration_tests/test_dac/test_dac.py
@@ -71,6 +71,8 @@ def test_dac_write_and_read_voltages(analog_out, voltage, raises, dac):
     with raises:
         # expected_voltage = (voltage)*dac.dac_ops.dict_calib_param[analog_out.value].gain -\
         #  dac.dac_ops.dict_calib_param[analog_out.value].offset
+        # explicitly disable dac gain for this test
+        dac.set_dac_gain(False)
         dac.write_voltage(analog_out, voltage)
         code, voltage_val, gain_state = dac.get_state(analog_out, True, True, True)
         dac_gain = 2 if gain_state else 1

--- a/src/test_edgepi/integration_tests/test_eeprom/test_eeprom.py
+++ b/src/test_edgepi/integration_tests/test_eeprom/test_eeprom.py
@@ -33,6 +33,8 @@ def fixture_test_eeprom():
                         ])
 # pylint: disable=protected-access
 def test__page_write_register(data, address, eeprom):
+    # reset user space to make sure init vals are set to 255
+    eeprom.reset_user_space()
     addrx = EdgePiMemoryInfo.USER_SPACE_START_BYTE.value + address
     with eeprom.i2c_open():
         initial_data = eeprom._EdgePiEEPROM__sequential_read(addrx,len(data))

--- a/tests/hardware_tests/test_eeprom_userspace.py
+++ b/tests/hardware_tests/test_eeprom_userspace.py
@@ -13,7 +13,7 @@ import os
 PATH = os.path.dirname(os.path.abspath(__file__))
 
 import pytest
-from edgepi.eeprom.eeprom_constants import EEPROMInfo
+from edgepi.eeprom.eeprom_constants import EEPROMInfo, EdgePiMemoryInfo
 from edgepi.eeprom.edgepi_eeprom import EdgePiEEPROM
 
 
@@ -34,11 +34,19 @@ def read_dummy_json(file_name: str):
 def test_reset_user_space(eeprom):
     reset_vals = []
     eeprom.reset_user_space()
-    num_of_pages = int(EEPROMInfo.NUM_OF_PAGE.value / 2)
-    for page in range(num_of_pages, EEPROMInfo.NUM_OF_PAGE.value):
-        # pylint: disable=line-too-long
-        # pylint: disable=protected-access
-        reset_vals += eeprom._EdgePiEEPROM__sequential_read(page * EEPROMInfo.PAGE_SIZE.value,EEPROMInfo.PAGE_SIZE.value)
+    num_of_pages = EdgePiMemoryInfo.USER_SPACE_END_PAGE.value - \
+                   EdgePiMemoryInfo.USER_SPACE_START_PAGE.value + 1
+    mem_offset = EdgePiMemoryInfo.USER_SPACE_START_BYTE.value
+
+    with eeprom.i2c_open():
+        for page in range(num_of_pages):
+            # NOTE: private seq_read is used to ignore CRC check!
+            # pylint: disable=protected-access
+            reset_vals += eeprom._EdgePiEEPROM__sequential_read(
+                    mem_offset + (page * EEPROMInfo.PAGE_SIZE.value),
+                    EEPROMInfo.PAGE_SIZE.value
+            )
+
     for val in reset_vals:
         assert val == 255
 

--- a/tests/hardware_tests/test_voltage_rw.py
+++ b/tests/hardware_tests/test_voltage_rw.py
@@ -134,7 +134,7 @@ def _measure_voltage_differential(
     for _ in range(READS_PER_WRITE):
         read_voltage = adc.read_voltage(adc_num)
         _logger.info(f"diff_read_voltage = {read_voltage}")
-        _assert_approx(write_volt/2, read_voltage, RW_ERROR)
+        _assert_approx(write_volt, read_voltage, RW_ERROR)
 
 
 _diff_ch_map = {

